src/styles/_components/_tooltip.scss

```diff
@@ -1,4 +1,4 @@
-@mixin tooltip() {
+@mixin tooltip($top : '0', $left : '28px') {
```

Comment:
TBH it's irrelevant because if not passed as param '0' will be used, else the passed param will be used (with or without units)